cv/lane_detection/src/lane_detection_wrapper_client.py

```diff
@@ -30,8 +30,8 @@ def run(self):
             lanes = self._fromRedisLanes('lane_detection')
             mask = self._fromRedisImg('cv/model/output')
 
+            img = self.bridge.cv2_to_imgmsg(mask*255, encoding='passthrough')
```

Comment:
@spencerteetaert 

Also we need a try and catch here because this can return an exception